templates/docs/api.html

```diff
@@ -156,12 +156,10 @@ <h3>{{tag}}</h3>
               {%endfor%}
             {%endif%}
             {%if "requestBody" in item[endpoint][method]%}
-              {%if "parameters" not in item[endpoint] and "parameters" not in item[endpoint][method]%}
-                <p><strong>Parameters</strong></p>
-                <hr />
-              {%endif%}
-              {%for property in item[endpoint][method]["requestBody"]["content"]["application/json"]["schema"]["properties"]%}
-                <p><md-block><code>{{property}}</code> (<em>{{item[endpoint][method]["requestBody"]["content"]["application/json"]["schema"]["properties"][property]["type"]}}</em>): {%if property in item[endpoint][method]["requestBody"]["content"]["application/json"]["schema"]["required"]%}Required{%else%}Optional{%endif%}. {{item[endpoint][method]["requestBody"]["content"]["application/json"]["schema"]["properties"][property]["description"].replace(" -", "\n -")}}</md-block></p>
+              <p><strong>Request body (multipart/form-data)</strong></p>
```

Comment:
Are you sure this is true for all `request_body`s or only the ones with files. Or to put it differently, can we check for `item[endpoint][method]["requestBody"]["content"]["multipart/form-data"]` directly?

Reply:
I think we can check for this, but currently I am sure that this is true for all request bodies - trying this with "application/json" results in errors, but having run this locally with the most recent schema, there are no errors when using "multipart/form-data". I've checked through the schema to verify this as well.